index.html

```diff
@@ -37,6 +37,177 @@ <h1 class="intro">I’m a software developer</h1>
             </button>
         </section>
     </section>
+    <section class="works_section">
+        <h1>My Recent <br> Works</h1>
```

Comment:
- [ ] The page is supposed to only have one heading element `h1` for semantic reasons, Kindly you may consider using `h2` ...`hx`